package.json

```diff
@@ -63,6 +63,7 @@
         "vue3-datepicker": "^0.2.4",
         "vue3-dropzone": "^0.0.7",
         "vue3-timepicker": "*",
-        "vuex": "^4.0.2"
+        "vuex": "^4.0.2",
+        "ziggy-js": "^2.3.0"
```

Comment:
Ziggy now has a JS package to to what the old import from `../vendor/ziggy/.....` used to do.

Reply:
There's been a couple updates to ziggy as inertia updated. I might have missed this one.